ArjunaJTA/jakartaee_transactional_app/README.md

```diff
@@ -9,6 +9,14 @@ bean and then test it using testsuite deployed in WildFly using Arquillian
 
 This command will build and test the application using Arquillian
 
+Get a WildFly that has the expected version of Narayana in it. If you need to build it:
+```
+WORKSPACE=<QUICKSTARTS_REPO_ROOT> ../../scripts/hudson/quickstart.sh clone_as
```

Comment:
What about changing this to 
`WORKSPACE=<QUICKSTARTS_REPO_ROOT> $WORKSPACE/scripts/hudson/quickstart.sh clone_as`

Reply:
I think the problem with it is that it will leave WORKSPACE set in the current environment in what I think is not super immediately obvious and so would then (IMO) need to be explained to the user (and probably need to add a suggestion to unset it WORKSPACE when finished). I am not "-1" to the idea, but I am not sure because the command as proposed is self contained.

Reply:
I am not super sure I like this but I wonder about:
`WORKSPACE=<QUICKSTARTS_REPO_ROOT> <QUICKSTARTS_REPO_ROOT>/scripts/hudson/quickstart.sh clone_as`

Reply:
IIRC if you only run "WORKSPACE=<QUICKSTARTS_REPO_ROOT>; $WORKSPACE/scripts/hudson/quickstart.sh clone_as" than you don't need to unset it. You only have to when exporting the variable. 
BTW  "WORKSPACE=<QUICKSTARTS_REPO_ROOT> <QUICKSTARTS_REPO_ROOT>/scripts/hudson/quickstart.sh clone_as" is fine too ;)

Reply:
I was composing a response but I see your update now :) will go with that and resolve this comment and then work on trying to roll this out more

Reply:
Thank you for the input!

Reply:
Thank you!